src/components/Dialogs/ActionDialog/ActionDialog.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import BaseDialog, { BaseDialogProps } from '../BaseDialog'
+import { ActionsContainer, ButtonsContainer, AdditionalActionsContainer } from './ActionDialog.style'
+import { Button } from '@/shared/components'
+
+export type ActionDialogProps = {
+  additionalActionsNode?: React.ReactNode
+  primaryButtonText?: string
```

Comment:
Buttons will probably need to be nodes instead, I'll need `disabled` as well and that seems like passing too many props.